fastly/fixtures/sftps/update.yaml

```diff
@@ -2,31 +2,34 @@
+      PRIVATE KEY-----\n","placement":"waf_debug","message_type":"classic","created_at":"2020-06-03T13:57:30Z","password":"password"}'
```

Comment:
This line includes the updated `message_type` value (i.e., "blank" -> "classic").